### ISSUE TITLE
Release 2.x ?

### ISSUE THREAD

**Issue by xoco70** [Altoros/Ansible-Fabric-Starter#11]
Will you update the script for HLF 2.x ?

**Comment by yod527** [comment]
Yes, but definitely not in the nearest future, so feel free to contribute!

**Comment by agustinustheo** [comment]
Hey @yod527, I've tried upgrading the Fabric version to version `2.2.2` here: https://github.com/agustinustheo/ansible-fabric-starter/tree/v2

I'll make a PR soon when I've tested the chaincode.

**Comment by agustinustheo** [comment]
I made my PR here https://github.com/Altoros/Ansible-Fabric-Starter/pull/24

If you're still maintaining this, please review and merge when appropriate.

**Comment by agustinustheo** [comment]
Thanks @yod527
